mesatee_services/tdfs/sgx_trusted_lib/src/tdfs_external.rs

```diff
@@ -135,8 +145,13 @@ impl HandleRequest for DeleteFileRequest {
 
         let target = config::Internal::target_kms();
         let mut client = KMSClient::new(target)?;
-        let key_resp = client.request_del_key(&file_meta.key_id)?;
-        let key_config = key_resp.config;
+        let key_req = kms_proto::proto::DeleteKeyRequest::new(&file_meta.key_id);
+        let key_resp = client.del_key(key_req)?;
+        let key_config = key_resp.get_key_config()?;
+        let key_config = match key_config {
+            kms_proto::KeyConfig::Aead(config) => kms_proto::proto::AeadConfig::from(config),
+            kms_proto::KeyConfig::ProtectedFs(_config) => unimplemented!(), // ProtectedFS is not used by TDFS yet. Config of ProtectedFs will not be generated neither.
```

Comment:
shall we return an error here?

Reply:
At this time, this code should be unreachable. 
Adding a new Error type also makes sense.

Reply:
How about ErrorKind::UnreachableCode

Reply:
No...I don't think this should be an error. Since this is just a placeholder and will be implemented anyway, otherwise the tdfs is not complete.

Reply:
Okay. I will keep the ```unimplemented```